website/src/comment-panel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {SetStateAction, ReactNode } from "react"
+import * as Dailp from "src/graphql/dailp"
+import { TranslatedParagraph } from "./segment"
+import { useState } from "react"
+import { Button } from "./components"
+import * as css from "./comment-panel.css"
+
+export const CommentPanel = (p: {
+    word: Dailp.FormFieldsFragment | null
+    segment: TranslatedParagraph | null
+    setCommentsPanel: React.Dispatch<SetStateAction<boolean>>
+}) => {
+    const [newCommentText, setNewCommentText] = useState<string>('')
+    const [newCommentType, setNewCommentType] = useState('')
+
+    // Prob messing up smth here
+    const [postCommentResult, postComment] = Dailp.usePostCommentMutation()
+
+    const options = [
+        // There is def a better way to do this..
```

Comment:
Is there a better way to do this than hard-coding it? Probably, right?

Reply:
see https://github.com/NEU-DSG/dailp-encoding/pull/361#discussion_r1414326461